### PR TITLE
docs: Update websites README.md

### PR DESCRIPTION
**Pull request by darcywong00**
Update website-local-proxy names

@keymanapp-test-bot skip

### CODE REVIEW

File: docs/websites/README.md [hunk 1]
```diff
@@ -54,10 +60,13 @@ The link file also appears locally.
...
-| Website      |          URL          |
-|--------------|-----------------------|
-|help.keyman   | http://localhost:8055 |
-|keymanweb.com | http://localhost:8057 |
+| Website        |          URL          |  with website-local-proxy running |
+|----------------|-----------------------|-----------------------------------|
+| keyman.com     | http://localhost:8053 | http://keyman.com.localhost       |
+| s.keyman.com   | http://localhost:8054 | http://s.keyman.com.localhost     |
+| help.keyman    | http://localhost:8055 | http://help.keyman.com.localhost  |
+| keymanweb.com  | http://localhost:8057 | http://keymanweb.com.localhost    |
```
Comment (mcdurdin) [inline docs/websites/README.md:68]: does web.keyman.com.localhost also work?

**Review by mcdurdin — APPROVED** [review]

File: docs/websites/README.md (hunk 1 above)
Comment (darcywong00) [reply docs/websites/README.md:68]: err, confusingly that link gets me api.keyman.  I'll have to investigate why...

Comment (mcdurdin) [reply docs/websites/README.md:68]: hmm, that does sound like it needs to be fixed!

Comment (darcywong00) [reply docs/websites/README.md:68]: Added in keymanapp/website-local-proxy#6

Comment (darcywong00) [reply docs/websites/README.md:68]:
```suggestion
| keymanweb.com  | http://localhost:8057 | http://keymanweb.com.localhost    |
|                                  |                                       | http://web.keyman.com.localhost   |

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.257-alpha](https://keyman.com/downloads/releases/alpha/17.0.257)

**Pull request merged**